css/style.css

```diff
@@ -82,6 +82,7 @@ body {
 .prev, .next {
   cursor: pointer;
   position: absolute;
+  z-index: 1;
```

Comment:
keep the arrows on top of the slides, which are now being appended _after_ the arrows in the DOM.